modules/porous_flow/doc/content/modules/porous_flow/solute_tracer_transport.md

```diff
@@ -0,0 +1,153 @@
+
+!listing modules/porous_flow/examples/solute_tracer_transport/solute_tracer_transport.i block=AuxVariables
+
+!listing modules/porous_flow/examples/solute_tracer_transport/solute_tracer_transport.i block=AuxKernels
+
+### Postprocessor
+
+As previously discussed, we only want to know the variation of concentration (C) and x-direction Darcy velocity at x=50 m over time. So, we need to tell MOOSE to only write that information into the result file via this code block.
+
+!listing modules/porous_flow/examples/solute_tracer_transport/solute_tracer_transport.i block=Postprocessors
+
+## Result
+
+The results obtained from MOOSE are compared against the analytical solution proposed by [!citep](Ogata1961ASO).
+
+!media porous_flow/concen_time.png caption= Tracer concentration with respect to time.  id=concen_time
+
+!media porous_flow/RMS_tracer.png caption= Root-mean-square error of MOOSE results.  id=RMS_tracer
+
+Compared with the benchmark conducted by [!cite](w14081310), it can be seen that MOOSE is capable of delivering accurate results for this type of problem.
+
+!media porous_flow/benchmark_tracer.png caption= The benchmark conducted by [!citep](w14081310).  id=benchmark_tracer
+
+## 2D solute transportation
+
+## Problem statement
+
+This example is based on [!citep](w14081310). The transportation of tracer in water in a 2D porous tank is modelled. The schematic is demonstrated in [!ref](schematic_tracer_2D). The parameters are the same as the 1D problem as provided in [!ref](datatable).
+
+!media porous_flow/schematic_tracer_2D.png caption=The schematic of the 2D problem [!citep](w14081310).  id=schematic_tracer_2D
+
+## Input file setup
+
+The input file for this problem is similar to the 1D case except for some changes made to account for the 2D mesh and the changes in the boundary conditions which are elaborated in the following sections. Other sections will be the same as the 1D case.
+
+### Mesh
+
+For this problem, to simulate a 2D rectangular aquifer  with a point source placed at the origin, the x-domain was chosen from -50 to 50 m and the y-domain from 0 to 50 m.
+
+!listing modules/porous_flow/examples/solute_tracer_transport/solute_tracer_transport_2D.i block=Mesh
+
+### DiracKernels set-up
+
+Since our problem comprises a point source, PorousFlowSquarePulsePointSource DiracKernels are required [(Details can be found here)](https://mooseframework.inl.gov/moose/source/dirackernels/PorousFlowSquarePulsePointSource.html). These DiracKernels  will act as the inlet boundary condition which supplies the water and tracer into the domain. As can be seen from the code packet below, two DiracKernels are being used. The first one supplies the water with a specified mass flux to the system and the second one for the tracer.
```

Comment:
```suggestion
Since our problem comprises a point source, [`PorousFlowSquarePulsePointSource'](/porous_flow/PorousFlowSquarePulsePointSource.md) DiracKernels are required. These DiracKernels inject the water and tracer into the domain at a constant rate. As can be seen below, two DiracKernels are used. The first one supplies the water with a specified mass flux to the system and the second one the tracer.
```

Reply:
filename.md only seems work, let see if some errors occur when I push it on.